network/p2p/p2pnode/libp2pStream_test.go

```diff
@@ -613,82 +550,3 @@ func TestCreateStreamIsConcurrent(t *testing.T) {
 		"CreateStream attempt to the unresponsive peer did not block after connecting to good node")
 
 }
-
-// TestConnectionGating tests node allow listing by peer.ID
-func TestConnectionGating(t *testing.T) {
```

Comment:
Reason for removal: this test helper logic has been moved to `connectionGater_test.go`, generalized and revamped (see [here](https://github.com/onflow/flow-go/blob/2c5d063e6a041bf84a2275288e79a6b76d5d1363/network/p2p/connection/connectionGater_test.go#L28))